core/domain/exp_services.py

```diff
@@ -378,7 +378,12 @@ def apply_change_list(exploration_id, change_list):
                 elif (
                         change.property_name ==
                         exp_domain.STATE_PROPERTY_INTERACTION_SOLUTION):
-                    state.update_interaction_solution(change.new_value)
+                    solution = None
+                    if change.new_value:
```

Comment:
if change.new_value is not None (Pythonic) [If we just want to pass this for "not None" case]

Reply:
Done